src/eredis.erl

```diff
@@ -27,15 +27,52 @@
 %% Exported for testing
 -export([create_multibulk/1]).
 
+-type transport() :: tcp | ssl.
 -export_type([transport/0]).
+
+-type reconnect_sleep() :: no_reconnect | integer().
+
+-type option() ::
+        {transport, transport()} |
+        {host, string()} |
+        {host, {local, term()}} |
+        {port, integer()} |
+        {database, string()} |
+        {database, undefined} |
+        {password, string()} |
+        {reconnect_sleep, reconnect_sleep()} |
+        {connect_timeout, non_neg_integer()}.
+-type server_args() :: [option()].
```

Comment:
^ We should export this, since it's part of the API.

Reply:
Perhaps you fixed it later? Please resolve if that's the case. I'm sorry, I don't think this commit-by-commit approach to reviews works very well for me...

Reply:
This is exported.